### DISCUSSION TITLE
FEEDBACK

### DISCUSSION THREAD

**Discussion by rsobrinog** [emirand20/M07_UF4_practica20#31] [General]
@emirand20 @andrescarles @4Tipy 

Mirando settings.py de vuestro último commit, veo que no tenéis añadidio en **INSTALLED_APPS** 'rest_framework'. Eso implica que no estáis utilizando Django_Rest_framework.
Ahora mismo estáis haciendo un proyecto en DJango. Vigilad.

Seguid el tutorial de la página oficial de Django Rest Framework.

Os dejo una imágen de cómo tenéis que ver las consultas de información a BBDD con cada view usando @api_view.

![image](https://user-images.githubusercontent.com/108349158/236436650-5a696cd7-69ce-40f1-9471-9c06ebcf2af5.png)

**Comment by rsobrinog** [comment]
@emirand20 @andrescarles @4Tipy 
Éste repositorio no lo borréis para nada. Sino os tendré que poner nota como si empezárais el 4 de mayo (y no os conviene).
Se que me comentastéis alfgo en clase (si no recuerdo mal) pero os pido que me indiquéis por éste feedback el porqué del cambio de repositorio.

**Comment by emirand20** [comment]
Queremos hacer mas limpio, por eso hemos creado otro repositorio.
